include/aws/crt/auth/Credentials.h

```diff
@@ -6,6 +6,8 @@
 
 #include <aws/crt/Exports.h>
 #include <aws/crt/Types.h>
+#include <aws/crt/http/HttpConnection.h>
```

Comment:
I don’t see where this include is being used.

Reply:
Proxy Options is there